src/Materials/effect.ts

```diff
@@ -325,6 +338,9 @@ export class Effect implements IDisposable {
 
         };
         this._loadShader(vertexSource, "Vertex", "", (vertexCode) => {
+            if (processorOptions.processor?.removeCommentsBeforeProcessing) {

```

Comment:
Why adding this here and not in the processor ?

Reply:
Code moved to `ShaderProcessor`